tests/test_settings.py

```diff
@@ -3,12 +3,12 @@
 from unittest import mock
 
 from screenpy import settings as screenpy_settings
-from screenpy.narration.stdout_adapter import settings as stdout_adapter_settings
 from screenpy.configuration import (
+    ScreenPySettings,
```

Comment:
This change was an example of isort vs ruff.isort.

Reply:
I think it might have been that `isort` wasn't run against this, 'cause this is how `isort` and ruff's `isort` both would have organized these.

Imports from `screenpy.configuration` should come before `screenpy.narration`, and class imports should come before function imports.

Reply:
I ran ruff against the file and it left the previous commit alone. /shrug

Reply:
`isort` isn't turned on through `ruff` yet—i'm working on it! 😩